shared/gh/partials/admin-batch-edit-event-row.html

```diff
@@ -3,20 +3,29 @@
 <% } else { %>
 <tr data-tempid="<%= data.ev.tempId %>" class="<% if (data.ev.selected) { %>info<% } %> active gh-new-event-row">
 <% } %>
+    <!-- Event select -->
     <td>
-        <div class="checkbox">
-            <label>
-                <input type="checkbox" class="gh-select-single" title="Click to select the event" <% if (data.ev.selected) { %> checked="checked" <% } %>> <span class="sr-only">Select</span>
-            </label>
-        </div>
+        <% if (!data.borrowedFrom) { %>
```

Comment:
It'd be nice if we could avoid rendering this `td` element when in read-only mode